src/views/router_views/psc_type/psc_type.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% block main_content %}
+
+{% set pscTypeTitle = i18n.psc_type_title %}
+{% set pscTypeIndividual = i18n.psc_type_individual %}
+{% set pscTypeRle = i18n.psc_type_rle %}
+{% set pscTypeHint = i18n.psc_type_hint | safe %}
+{% set detailsTitleText = i18n.psc_type_details_title %}
+{% set detailsText = i18n.psc_type_details_text %}
+{% set continueText = i18n.global_confirm_continue_button %}
+
+<p class="govuk-body govuk-!-margin-bottom-7"></p>
+
+{{ govukRadios({
+  name: "pscType",
+  fieldset: {
+    legend: {
+      text: pscTypeTitle,
+      isPageHeading: true,
+      classes: "govuk-fieldset__legend--l"
+    }
+  },
+  items: [
+    {
+      value: "individual",
+      text: pscTypeIndividual
+    },
+    {
+      value: "rle",
+      text: pscTypeRle,
+       hint: {
+        text: pscTypeHint
+      }
+    }
+  ]
+}) }}
+
+{{ govukDetails({
+summaryText: detailsTitleText,
+text: detailsText
+}) }}
+
+{{ govukButton({
+  text: continueText,
```

Comment:
You can replace continueText with the relevant i18n.xxx instead of creating a new variable

Reply:
BTW the text for the button is just continue rather than confirm and continue so you will probably need to create a new global key for that as other pages will also need it

Reply:
👍